### PR TITLE
Improvement for #293

### PR DESCRIPTION
**Pull request by thorsten**
As requested.

### CODE REVIEW

File: src/translate.service.ts [hunk 1]
```diff
@@ -396,13 +396,12 @@ export class TranslateService {
      * @returns string
      */
     public getBrowserLang(): string {
-        if(typeof window === 'undefined' || typeof window.navigator === 'undefined') {
+        let browserLang = this.getBrowserCultureLang();
+
+        if(browserLang === 'undefined') {
```
Comment (SamVerschueren) [inline src/translate.service.ts]: You mean `browserLang === undefined`?

Comment (ocombe) [inline src/translate.service.ts]: I'd rather keep `typeof window === 'undefined'` because undefined can be defined (yeah, I know :D)

Comment (SamVerschueren) [inline src/translate.service.ts]: I was referring to the browserLang check, it doesn't use `typeof`. So probably it should be changed to

```js
if(typeof browserLang === 'undefined') {
```

Comment (thorsten) [inline src/translate.service.ts]: You're right, it would be more consistent.

**Pull request closed without merging**